holidays/holiday_base.py

```diff
@@ -223,6 +227,7 @@ def __init__(
         prov: Optional[str] = None,  # Deprecated.
         state: Optional[str] = None,  # Deprecated.
         language: Optional[str] = None,
+        categories: Optional[Tuple[str]] = None,
```

Comment:
This needs to be reflected as a :param below.

Reply:
Done.